client/ionic/src/pages/SplashCarousel.tsx

```diff
@@ -1,48 +1,57 @@
 import React from 'react';
-import { FlowLoader, Flow, LoadedFlow } from '../content/flow';
 import {
   IonContent,
   IonPage,
   IonSlides,
   IonSlide,
   IonCard,
   IonCardContent,
+  IonCardHeader,
+  IonCardSubtitle,
+  IonCardTitle,
+  IonButton,
   IonImg,
 } from '@ionic/react';
-import TopNav from '../components/TopNav';
 import 'tachyons';
-import { getUserContext } from '../content/userContext';
 import useDynamicFlow from '../hooks/useDynamicFlow';
+import TopNav from '../components/TopNav';
+
+import { S } from '../i18n/S';
 
-const CopingEveryone: React.FC = () => {
+const Splash: React.FC = () => {
```

Comment:
```suggestion
const SplashCarousel: React.FC = () => {
```

let's keep the export name = file names for components using default export

Reply:
done